testcafe/tests/pages/photos-album-model.js

```diff
@@ -0,0 +1,273 @@
+  isExistingAndVisibile
+} from '../helpers/utils'
+import PhotoPage from '../pages/photos-model'
+const photoPage = new PhotoPage()
+
+export default class Page {
+  constructor() {
+    this.albumEmptyText = photoPage.folderEmpty
+      .child('h2')
+      .withText("You don't have any album yet") //!FIXME
+
+    this.albumContentWrapper = getElementWithTestId('album-pho-content-wrapper')
+    this.albumTitle = getElementWithTestId('pho-content-title')
+
+    // Album list
+    this.toolbarAlbumsList = getElementWithTestId('pho-toolbar-albums')
+    this.btnNewAlbum = getElementWithTestId('album-add')
+    this.album = albumName => {
+      return getElementWithTestId('pho-album').withAttribute(
+        'data-test-name',
+        albumName
+      )
+    }
+    this.albumDescription = albumName => {
+      return this.album(albumName)
+        .child('a')
+        .child('h4')
+    }
+
+    //New albums & edit album
+    this.inputAlbumName = getElementWithTestId('input-album-name')
+    this.pickerAlbumName = getElementWithTestId('pho-picker-album-name')
+    this.btnValidateAlbum = getElementWithTestId('validate-album') //Seme button for create album and Add to album
+    this.photoThumb = value => {
+      return Selector('[class*="pho-photo-item"]').nth(value)
+    }
+    this.photoSectionAddToAlbum = getElementWithTestId('picker-panel')
+      .child()
+      .child()
+      .withAttribute('data-test-id', 'photo-section')
+
+    this.allPhotosAddToAlbum = this.photoSectionAddToAlbum
+      .find('img')
+      .withAttribute('data-test-id', 'pho-photo-item')
+    this.photoCheckbox = this.photoSectionAddToAlbum.find(
+      '[class*="pho-photo-select"][data-input="checkbox"]'
+    )
+
+    //Inside an album
+    this.mainContent = Selector('[class*="pho-content"]')
+    this.toolbarAlbum = getElementWithTestId('pho-toolbar-album')
+    this.moreMenuAlbum = getElementWithTestId('more-btn-album')
+    this.moreMenuDownloadAlbum = getElementWithTestId('menu-download-album')
+    this.moreMenuRenameAlbum = getElementWithTestId('menu-rename-album')
+    this.moreMenuAddPhotosToAlbum = getElementWithTestId(
+      'menu-add-photos-to-album'
+    )
+    this.moreMenuDeleteAlbum = getElementWithTestId('menu-delete-album')
+    this.modalAlert = Selector('[class*="c-alert-wrapper"]')
+    this.btnBackToAlbum = getElementWithTestId('pho-content-album-previous')
+  }
+
+  async waitForLoading() {
+    await t.expect(photoPage.loading.exists).notOk('Page still loading')
+    await isExistingAndVisibile(this.albumContentWrapper, 'Content Wrapper')
+  }
+
+  //@param {string} when : text for console.log
+  async getPhotosToAddCount(when) {
+    let allPhotosCount
+    isExistingAndVisibile(this.photoSectionAddToAlbum, 'Photo section')
+
+    // await isExistingAndVisibile(this.photoAddToAlbumWrapper, 'Picture wrapper')
+    await isExistingAndVisibile(this.allPhotosAddToAlbum, 'Photo item(s)')
+    allPhotosCount = await this.allPhotosAddToAlbum.count
```

Comment:
Tu peux supprimer le `let allPhotosCount` et juste faire un : 

`const allPhotosCount = await this.allPhotosAddToAlbum.count`